gen/dcos-config.yaml

```diff
@@ -512,6 +512,101 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
+  - path: /etc/dcos-3dt-runner-config.json
+    content: |
+      {
+        "cluster_checks": {},
+        "node_checks": {
+          "checks": {
+            "components_health_master": {
+              "description": "Validate all components are healthy on the node. Using 3dt HTTP",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "--role", "master", "components"],
+              "timeout": "3s",
+              "roles": ["master"]
+            },
+            "components_health_agent": {
+              "description": "Validate all components are healthy on the node. Using 3dt HTTP",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "--role", "agent", "components"],
+              "timeout": "3s",
+              "roles": ["agent"]
+            },
+            "executable_xz": {
+              "description": "find if xz is available",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "executable", "xz"],
+              "timeout": "1s"
+            },
+            "executable_wget": {
+              "description": "find if wget is available",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "executable", "wget"],
+              "timeout": "1s"
+            },
+            "executable_tar": {
+              "description": "find if tar is available",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "executable", "tar"],
+              "timeout": "1s"
+            },
+            "executable_curl": {
+              "description": "find if curl is available",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "executable", "curl"],
+              "timeout": "1s"
+            },
+            "executable_git": {
+              "description": "find if git is available",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "executable", "git"],
+              "timeout": "1s"
+            },
+            "executable_unzip": {
+              "description": "find if unzip is available",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "executable", "unzip"],
+              "timeout": "1s"
+            },
+            "check_ip": {
+              "description": "validate detect_ip script output",
+              "cmd": ["/opt/mesosphere/bin/dcos-checks", "ip"],
+              "timeout": "1s"
+            },
+            "check_mesos_metrics_master": {
+              "description": "check if mesos replog is synchronized",
```

Comment:
The description is fine, we can change the name of the file as a separate commit, @branden you ok with that?